_migrations/migration-phases/backfill.md

```diff
@@ -0,0 +1,174 @@
+
+   ```shell
+   console kafka describe-topic-records
+   ```
+   
+   Note the records in the logging topic.
+   
+2. After a short period, execute the same command again and compare the increase in records against the expected HTTP requests.
+
+
+## Creating a snapshot
+
+Create a snapshot for your backfill using the following command:
+
+```bash
+console snapshot create
+```
+
+To check the progress of your snapshot in realtime, use the following command:
+
+```bash
+console snapshot status --deep-check
+```
+
+Depending on the size of the data on the source cluster and the bandwidth allocated for snapshots, the process can take some time. Adjust the maximum rate at which the source cluster's nodes create the snapshot using the `--max-snapshot-rate-mb-per-node` option. Increasing the snapshot rate will consume more node resources, which may affect the cluster's ability to handle normal traffic.
+
+## Backfilling documents to the source cluster
+
+From the snapshot you created of your source cluster, you can begin backfilling documents to the target cluster. Once starting this process, a fleet of workers will spin up to read the snapshot and reindex documents on the target cluster. This fleet of workers can be scaled to increased the speed that documents are reindexed onto the target cluster.
+
+### Checking the starting state of the clusters
+
+You can see the indices and rough document counts of the source and target cluster by running the cat-indices command.  This can be used to monitor the difference between the source and target for any migration scenario.  Check the indices of both clusters with the following command:
+
+```shell
+console clusters cat-indices
+```
+
+You should receive the following response:
+
+```shell
+SOURCE CLUSTER
+health status index       uuid                   pri rep docs.count docs.deleted store.size pri.store.size
+green  open   my-index WJPVdHNyQ1KMKol84Cy72Q   1   0          8            0     44.7kb         44.7kb
+
+TARGET CLUSTER
+health status index                        uuid                   pri rep docs.count docs.deleted store.size pri.store.size
+green  open   .opendistro_security         N3uy88FGT9eAO7FTbLqqqA   1   0         10            0     78.3kb         78.3kb
+```
+
+### Starting the backfill
+
+Use the following command to start the backfill and deploy the workers:
+
+```shell
+console backfill start
+```
+
+You should receive a response similar to the following:
+
+```shell
+BackfillStatus.RUNNING
+Running=1
+Pending=0
+Desired=1
+Shards total: 48
+Shards completed: 48
+Shards incomplete: 0
+Shards in progress: 0
+Shards unclaimed: 0
+```
+
+The status will list as `Running` even if all the shards have been migrated.
+
+
+### Scaling up the fleet
+
+To speed up the transfer, you can scale the number of workers. It may take a few minutes for these additional workers to come online.  The following command will update the worker fleet to a size of ten:
```

Comment:
```suggestion
To speed up the transfer, you can scale the number of workers. It may take a few minutes for these additional workers to come online. The following command will update the worker fleet to a size of ten:
```